documentation/modules/exploit/multi/http/jenkins_script_console.md

```diff
@@ -0,0 +1,66 @@
+## Vulnerable Application
+
+  Jenkins can be downloaded from [jenkins.io](https://jenkins.io/) where
```

Comment:
There's not actually a vulnerable version, this module leverages a feature provided by the server.